transaction-service-api/src/main/java/pl/adamboguszewski/transactionservice/service/api/transaction/CreateTransactionResponse.java

```diff
@@ -0,0 +1,17 @@
+package pl.adamboguszewski.transactionservice.service.api.transaction;
+
+import lombok.NonNull;
+import lombok.Value;
+
+import java.util.UUID;
+
+@Value
+public class CreateTransactionResponse {
```

Comment:
What if something goes wrong, for example database exploded and we couldn't make a connection? We need two response classes - one for successful transaction creation and one if something goes not yes.